site/topics/if-else/if-else.rst

```diff
@@ -1,45 +1,89 @@
-**************
-Logic, if/else
-**************
+*******
+If/Else
+*******
 
-Conditionals
-============
+.. admonition:: Activity
 
-* We're pretty good at telling python *exactly what to do* now.
-* But there is no room for choice. It's just: *do these steps in this order*
-* That doesn't give us much flexibility (or computational power)
+    Using only what we have learned so far, write a function ``smush(a_number)`` that checks if the number is
+    positive or negative. If ``a_number`` is positive, the function will return half the value of ``a_number``. If
+    the number is not positive, simply return ``a_number``.
 
 
+Conditional Statements
```

Comment:
statement -> expression. A statement is an action, like `if`/`else`, which may consist of several expressions. An expression, on the other hand, is something that evaluates to a value. A conditional expression is an expression which evaluates to a boolean value.

Reply:
I believe _technically_ all expressions are statements. Nevertheless, your point will be incorporated as, at the very least, is more correct.